staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss.go

```diff
@@ -992,10 +992,22 @@ func (ss *scaleSet) ensureVMSSInPool(service *v1.Service, nodes []*v1.Node, back
 		if err != nil {
 			return err
 		}
-		primaryIPConfig, err := getPrimaryIPConfigFromVMSSNetworkConfig(primaryNIC)
-		if err != nil {
-			return err
+		var primaryIPConfig *compute.VirtualMachineScaleSetIPConfiguration
+		ipv6 := utilnet.IsIPv6String(service.Spec.ClusterIP)
```

Comment:
more comments on this? why ipv6 and ipv4 get from different config?

Reply:
and there must be a issue related to this PR if it's going to cherry-pick to old release

Reply:
Thank you. Added a comment and opened an issue #93085